### ISSUE TITLE
Cannot Scroll or Jump to Bottom of Conversation with VoiceOver

### ISSUE THREAD

**Issue by Simon818** [TelegramMessenger/Telegram-iOS#1156]
### Checklist
- ☑ I am reporting an issue in existing functionality that does not work as intended
- ☑ I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
Describe the issue that you are experiencing

VoiceOver users cannot jump to the bottom of a conversation. In a chat with hundreds of unread messages, finding the end is time-consuming an dfrustrating.

### Expected Behavior
Tell us what should happen

I'm not sure how sighted users jump to the end of a chat so i'm not sure what to suggest as an expectatoin except "It should be way easier than it is." Is there a scrollbar? If so, we can't access it.

### Actual Behavior
Tell us what happens instead

Scrolling through messages has always been problematic and nothing has been done about it. Swiping left and right should systematically move through all controls on the screen, scrolling any scrollable views as it goes. This doesn't work reliably on Telegram. A three-finger swipe up should scroll down messages, but this is also not reliable because VoiceOver is not aware it is scrolling, and eventually I have to touch the screen to refocus VoiceOver on the newly-visible message before I can scroll again. Sometimes a three-finger scroll will jump several pages at a time for no discernable reason. There is no button or control to scroll quickly down the screen, whereas most apps with large scrolling views have a "vertical scroll bar" on the right side which can be touched and scrolled easily. Therefore, trying to read a long conversation continuously is very difficult, and trying to scroll past it is also very difficult.

### Steps to Reproduce
1. Open a conversation which has a lot of messages in it (at least 20 pages, preferably more.)
2. Scroll all the way back to the start of the chat.
3. Load VoiceOver. Try finding a way to get to the bottom of the chat without unloading VoiceOver and without repeatedly three-finger scrolling.
4. Touch a message in the conversation and then repeatedly swipe right. Eventually, VoiceOver will stop scrolling the messages automatically and might move to the controls below, or just stop navigating entirely. It will be necessary to three-finger scroll--which sometimes loses one's place entirely--in order to keep reading the chat history.

### Screen Recording

[Screen Recording on Dropbox](https://www.dropbox.com/scl/fi/7czdzu06e0y2yrfzzduvg/2023-07-26-23.37.44.mp4?rlkey=ruufgeuapm1ot88xzwalepekl&dl=0)

Sorry this is quite long (3:03)

### Environment

**Device:** iPhone 13

**iOS version**: 16.6

**App version:** 9.6.5
